deploy/install/deploy_wellcheck.sh

```diff
@@ -57,28 +57,48 @@ else
     exit 1
 fi
 
+manager_ip=$(kubectl get node $(kubectl get pods -o wide | grep zeta-manager | awk '{print $7}' | cut -d/ -f1) -o wide | grep kind | awk '{print $6}' | cut -d/ -f1)
+
+response=$(curl -H 'Content-Type: application/json' -X POST \
+    -d '{"zgc_id":"0",
+          "name":"zgc0",
+          "description":"zgc0",
+          "ip_start":"10.0.0.0",
+          "ip_end":"10.0.0.255",
+          "port_ibo":"1"}' \
```

Comment:
don't use 1 for ibo port, default use 8300

Reply:
Ok!